app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -57,9 +57,9 @@ public function __construct()
      */
     protected function validator(array $data)
     {
-        $data['captcha'] = $this->captchaCheck();
-
-        if (!config('settings.reCaptchStatus')) {
+        if (config('settings.reCaptchStatus')) {
+            $data['captcha'] = $this->captchaCheck();
+        } else {
```

Comment:
This `else` statement is superfluous and is not of the happy path methodology in place.